core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -80,6 +82,14 @@ public RocksBlockStore(String baseDir) {
         .setCreateMissingColumnFamilies(true);
     String dbPath = PathUtils.concatPath(baseDir, BLOCKS_DB_NAME);
     String backupPath = PathUtils.concatPath(baseDir, BLOCKS_DB_NAME + "-backups");
+    // Create block store db path if it does not exist.
+    if (!FileUtils.exists(dbPath)) {
+      try {
+        FileUtils.createDir(dbPath);
+      } catch (IOException ioe) {
```

Comment:
nit: unless `e` is taken, just name exception to be `e` in a `catch` clause

Reply:
Hmm. I didn't know about this convention. I might have broken it many times..